### PR TITLE
Fix various xref macro calls

### PR DESCRIPTION
**Pull request by queengooborg**
This PR fixes various `xref` macro calls throughout the documentation.

### CODE REVIEW

File: files/en-us/web/api/hidconnectionevent/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ browser-compat: api.HIDConnectionEvent
 
 {{securecontext_header}}{{APIRef("WebHID API")}}{{SeeCompatTable}}
 
-The **`HIDConnectionEvent`** interface of the {{domxref('WebHID API')}} represents HID connection events, and is the event type passed to {{domxref("HID.onconnect")}} and {{domxref("HID.ondisconnect")}} when an input report is received.
+The **`HIDConnectionEvent`** interface of the {{domxref('WebHID API')}} represents HID connection events, and is the event type passed to {{domxref("HID.connect_event")}} and {{domxref("HID.disconnect_event")}} when an input report is received.
```
Comment (Josh-Cena) [inline files/en-us/web/api/hidconnectionevent/index.md:12]:
```suggestion
The **`HIDConnectionEvent`** interface of the {{domxref('WebHID API')}} represents HID connection events, and is the event type passed to {{domxref("HID/connect_event", "connect")}} and {{domxref("HID/disconnect_event", "disconnect")}} event handlers when an input report is received.
```

**Review by Josh-Cena — DISMISSED** [review]
Could you make sure all the macros still display user-friendly text, apart from this one? Also, for the deprecated features, I'm not sure removing them is the right thing to do if they still exist; maybe we can just change it to plain text.

File: files/en-us/web/api/html_dom_api/microtask_guide/in_depth/index.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ There are three types of event loop:
 - Worker event loop
   - : A worker event loop is one which drives a worker; this includes all forms of workers, including basic [web workers](/en-US/docs/Web/API/Web_Workers_API), [shared workers](/en-US/docs/Web/API/SharedWorker), and [service workers](/en-US/docs/Web/API/Service_Worker_API). Workers are kept in one or more agents that are separate from the "main" code; the browser may use a single event loop for all of the workers of a given type or may use multiple event loops to handle them.
 - Worklet event loop
-  - : A [worklet](/en-US/docs/Web/API/Worklet) event loop is the event loop used to drive agents which run the code for the worklets for a given agent. This includes worklets of type {{domxref("Worklet")}}, {{domxref("AudioWorklet")}}, and {{domxref("PaintWorklet")}}.
+  - : A [worklet](/en-US/docs/Web/API/Worklet) event loop is the event loop used to drive agents which run the code for the worklets for a given agent. This includes worklets of type {{domxref("Worklet")}}, {{domxref("AudioWorklet")}}, and {{domxref("PaintWorkletGlobalScope")}}.
```
Comment (chrisdavidmills) [inline files/en-us/web/api/html_dom_api/microtask_guide/in_depth/index.md:107]: Paint worklets are a funny case where they actually just use a normal `Worklet` object to do specific tasks, as accessed by `CSS.paintWorklet`. `PaintWorkletGlobalScope` isn't right in this context; that refers to the top-level scope inside the worklet, and not the worklet itself. I'd just remove it altogether:

> ...worklets of type {{domxref("Worklet")}} and {{domxref("AudioWorklet")}}.

File: files/en-us/web/api/midiport/close/index.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ browser-compat: api.MIDIPort.close
 
 The **`close()`** method of the {{domxref("MIDIPort")}} interface makes the access to the MIDI device connected to this `MIDIPort` unavailable.
 
-If the port is successfully closed a new {{domxref("MIDIConnectionEvent")}} is queued to the {{domxref("MIDIPort.onstatechange")}} and {{domxref("MIDIAccess.onstatechange")}} event handlers, and the {{domxref("MIDIPort.connection")}} property is changed to `"closed"`.
+If the port is successfully closed a new {{domxref("MIDIConnectionEvent")}} is queued to the {{domxref("MIDIPort.statechange_event")}} and {{domxref("MIDIAccess.statechange_event")}} events, and the {{domxref("MIDIPort.connection")}} property is changed to `"closed"`.
```
Comment (chrisdavidmills) [inline files/en-us/web/api/midiport/close/index.md:13]:
```suggestion
If the port is successfully closed a new {{domxref("MIDIConnectionEvent")}} is queued to the {{domxref("MIDIPort.statechange_event", "MIDIPort statechange")}} and {{domxref("MIDIAccess.statechange_event", "MIDIAccess statechange")}} events, and the {{domxref("MIDIPort.connection")}} property is changed to `"closed"`.
```

? Bit of a difficult one to gauge how to write.

File: files/en-us/web/api/midiport/open/index.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ browser-compat: api.MIDIPort.open
 
 The **`open()`** method of the {{domxref("MIDIPort")}} interface makes the MIDI device connected to this `MIDIPort` explicitly available.
 
-If the port is successfully opened a new {{domxref("MIDIConnectionEvent")}} is queued to the {{domxref("MIDIPort.onstatechange")}} and {{domxref("MIDIAccess.onstatechange")}} event handlers, and the {{domxref("MIDIPort.connection")}} property is changed to `"open"`.
+If the port is successfully opened a new {{domxref("MIDIConnectionEvent")}} is queued to the {{domxref("MIDIPort.statechange_event")}} and {{domxref("MIDIAccess.statechange_event")}} events, and the {{domxref("MIDIPort.connection")}} property is changed to `"open"`.
```
Comment (chrisdavidmills) [inline files/en-us/web/api/midiport/open/index.md:13]:
```suggestion
If the port is successfully opened a new {{domxref("MIDIConnectionEvent")}} is queued to the {{domxref("MIDIPort.statechange_event", "MIDIPort statechange")}} and {{domxref("MIDIAccess.statechange_event", "MIDIAccess statechange")}} events, and the {{domxref("MIDIPort.connection")}} property is changed to `"open"`.
```

File: files/en-us/web/api/oscillatornode/index.md [hunk 5]
```diff
@@ -63,9 +63,9 @@ _Inherits methods from its parent, {{domxref("AudioScheduledSourceNode")}}, and
 
 - {{domxref("OscillatorNode.setPeriodicWave()")}}
   - : Sets a {{domxref("PeriodicWave")}} which describes a periodic waveform to be used instead of one of the standard waveforms; calling this sets the `type` to `custom`.
-- {{domxref("OscillatorNode.start()")}}
+- {{domxref("AudioScheduledSourceNode.start")}}
```
Comment (chrisdavidmills) [inline files/en-us/web/api/oscillatornode/index.md:66]: Should still have the parens, as it's a method?

File: files/en-us/web/api/oscillatornode/index.md [hunk 6]
```diff
@@ -63,9 +63,9 @@ _Inherits methods from its parent, {{domxref("AudioScheduledSourceNode")}}, and
...
   - : Specifies the exact time to start playing the tone.
-- {{domxref("OscillatorNode.stop()")}}
+- {{domxref("AudioScheduledSourceNode.stop")}}
```
Comment (chrisdavidmills) [inline files/en-us/web/api/oscillatornode/index.md:68]: Should still have the parens, as it's a method?

File: files/en-us/web/api/usbconnectionevent/index.md [hunk 7]
```diff
@@ -9,7 +9,7 @@ browser-compat: api.USBConnectionEvent
 
 {{securecontext_header}}{{APIRef("WebUSB API")}}{{SeeCompatTable}}
 
-The **`USBConnectionEvent`** interface of the {{domxref('WebUSB API','','',' ')}} is the event type passed to {{domxref("USB.onconnect")}} and {{domxref("USB.ondisconnect")}} when the user agent detects that a new USB device has been connected or disconnected.
+The **`USBConnectionEvent`** interface of the {{domxref('WebUSB API','','',' ')}} is the event type passed to {{domxref("USB.connect_event")}} and {{domxref("USB.disconnect_event")}} when the user agent detects that a new USB device has been connected or disconnected.
```
Comment (chrisdavidmills) [inline files/en-us/web/api/usbconnectionevent/index.md:12]:
```suggestion
The **`USBConnectionEvent`** interface of the {{domxref('WebUSB API','','',' ')}} is the event type passed to `USB` {{domxref("USB.connect_event", "connect")}} and {{domxref("USB.disconnect_event", "disconnect")}} events when the user agent detects that a new USB device has been connected or disconnected.
```

File: files/en-us/web/api/webhid_api/index.md [hunk 8]
```diff
@@ -20,7 +20,7 @@ A Human Interface Device (HID) is a type of device that takes input from or prov
 - {{domxref("HIDInputReportEvent")}}
   - : Passed to {{domxref("HIDDevice.inputreport_event")}} when an input report is received from any associated HID device.
 - {{domxref("HIDConnectionEvent")}}
-  - : Passed to {{domxref("HID.onconnect")}} and {{domxref("HID.ondisconnect")}} when a device is connected or disconnected.
+  - : Passed to {{domxref("HID.connect_event")}} and {{domxref("HID.disconnect_event")}} when a device is connected or disconnected.
```
Comment (chrisdavidmills) [inline files/en-us/web/api/webhid_api/index.md:23]:
```suggestion
  - : Passed to `HID` {{domxref("HID.connect_event", "connect")}} and {{domxref("HID.disconnect_event", "disconnect")}} events when a device is connected or disconnected.
```

File: files/en-us/web/api/webusb_api/index.md [hunk 9]
```diff
@@ -26,7 +26,7 @@ When connecting a new WebUSB-compatible device, the browser displays a notificat
 - {{domxref("USB")}}
   - : Provides attributes and methods for finding and connecting USB devices from a web page.
 - {{domxref("USBConnectionEvent")}}
-  - : The event type passed to {{domxref("USB.onconnect")}} or {{domxref("USB.ondisconnect")}} when the user agent detects a new USB device has been connected to, or disconnected from the host.
+  - : The event type passed to {{domxref("USB.connect_event")}} or {{domxref("USB.disconnect_event")}} when the user agent detects a new USB device has been connected to, or disconnected from the host.
```
Comment (chrisdavidmills) [inline files/en-us/web/api/webusb_api/index.md:29]:
```suggestion
  - : The event type passed to `USB` {{domxref("USB.connect_event", "connect")}} or {{domxref("USB.disconnect_event", "disconnect")}} events when the user agent detects a new USB device has been connected to, or disconnected from the host.
```

File: files/en-us/web/api/workletglobalscope/index.md [hunk 10]
```diff
@@ -9,7 +9,7 @@ browser-compat: api.WorkletGlobalScope
 
 The **`WorkletGlobalScope`** interface is an abstract class that specific worklet scope classes inherit from. Each `WorkletGlobalScope` defines a new global environment.
 
-> **Note:** You don't normally need to interact with this interface. It is a base interface intended to be subclassed. You will encounter the subclasses {{domxref("AudioWorkletGlobalScope")}} inside {{domxref("AudioWorklet")}} objects, or {{domxref("PaintWorkletGlobalScope")}} inside {{domxref("PaintWorklet")}} objects.
+> **Note:** You don't normally need to interact with this interface. It is a base interface intended to be subclassed. You will encounter the subclasses {{domxref("AudioWorkletGlobalScope")}} inside {{domxref("AudioWorklet")}} objects, or {{domxref("PaintWorkletGlobalScope")}} inside `PaintWorklet` objects.
```
Comment (chrisdavidmills) [inline files/en-us/web/api/workletglobalscope/index.md:12]:
```suggestion
> **Note:** You don't normally need to interact with this interface. It is a base interface intended to be subclassed. You will encounter the subclasses {{domxref("AudioWorkletGlobalScope")}} inside {{domxref("AudioWorklet")}} objects, or {{domxref("PaintWorkletGlobalScope")}} inside CSS paint {{domxref("Worklet")}} objects.
```

Again, paint worklets just use normal `Worklet`s.

File: files/en-us/web/css/@counter-style/additive-symbols/index.md [hunk 11]
```diff
@@ -7,7 +7,7 @@ browser-compat: css.at-rules.counter-style.additive-symbols
 
 {{CSSRef}}
 
-The **`additive-symbols`** descriptor lets you specify symbols when the value of a counter {{cssxref('system')}} descriptor is `additive`. The `additive-symbols` descriptor defines _additive tuples_, each of which is a pair containing a symbol and a non-negative integer weight. The additive system is used to construct [sign-value numbering](https://en.wikipedia.org/wiki/Sign-value_notation) systems such as Roman numerals.
+The **`additive-symbols`** descriptor lets you specify symbols when the value of a counter {{cssxref('@counter-style/system')}} descriptor is `additive`. The `additive-symbols` descriptor defines _additive tuples_, each of which is a pair containing a symbol and a non-negative integer weight. The additive system is used to construct [sign-value numbering](https://en.wikipedia.org/wiki/Sign-value_notation) systems such as Roman numerals.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@counter-style/additive-symbols/index.md:10]:
```suggestion
The **`additive-symbols`** descriptor lets you specify symbols when the value of a `@counter-style` {{cssxref('@counter-style/system', 'system')}} descriptor is `additive`. The `additive-symbols` descriptor defines _additive tuples_, each of which is a pair containing a symbol and a non-negative integer weight. The additive system is used to construct [sign-value numbering](https://en.wikipedia.org/wiki/Sign-value_notation) systems such as Roman numerals.
```

File: files/en-us/web/css/css_houdini/index.md [hunk 12]
```diff
@@ -40,7 +40,7 @@ Using {{cssxref("@property")}} however, we can declare the custom property with
 
 ## Houdini worklets
 
-A feature of Houdini is the {{domxref("Worklet")}}. A worklet is a module, written in JavaScript, that extends CSS using one of the Houdini APIs. You can see an example worklet on the {{domxref("PaintWorklet.registerPaint()")}} page. Once a worklet has been registered you can use it in CSS just like any other value. This means that even if you are not a JavaScript developer, you can access Houdini APIs by using worklets other people have created.
+A feature of Houdini is the {{domxref("Worklet")}}. A worklet is a module, written in JavaScript, that extends CSS using one of the Houdini APIs. You can see an example worklet on the {{domxref("PaintWorkletGlobalScope.registerPaint")}} page. Once a worklet has been registered you can use it in CSS just like any other value. This means that even if you are not a JavaScript developer, you can access Houdini APIs by using worklets other people have created.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/css_houdini/index.md:43]:
```suggestion
A feature of Houdini is the {{domxref("Worklet")}}. A worklet is a module, written in JavaScript, that extends CSS using one of the Houdini APIs. You can see an example worklet on the {{domxref("PaintWorkletGlobalScope.registerPaint()")}} page. Once a worklet has been registered you can use it in CSS just like any other value. This means that even if you are not a JavaScript developer, you can access Houdini APIs by using worklets other people have created.
```

File: files/en-us/web/css/hue-interpolation-method/index.md [hunk 13]
```diff
@@ -97,7 +97,7 @@ There are four algorithms to determine which arc is used:
 
 ### Comparing hue interpolation algorithms
 
-The following example shows the effect of using different hue interpolation algorithms in a {{CSSXref("linear-gradient()")}}.
+The following example shows the effect of using different hue interpolation algorithms in a {{CSSXref("gradient/linear-gradient")}}.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/hue-interpolation-method/index.md:100]:
```suggestion
The following example shows the effect of using different hue interpolation algorithms in a {{CSSXref("gradient/linear-gradient", "linear-gradient()")}}.
```

**Review by chrisdavidmills — APPROVED** [review]
@queengooborg A few more correction suggestions, but approving so as not to hold you up. Nice work!

**Pull request merged**